### PR TITLE
Add range filter to workQueue #338

### PR DESCRIPTION
**Pull request by Mikearaya**

### CODE REVIEW

File: packages/core-worker/director.js [hunk 1]
```diff
@@ -163,6 +163,8 @@ class WorkerDirector {
   static buildQueueSelector({
     status = [],
     selectTypes = [],
+    startDate,
+    endDate = new Date(),
```
Comment (pozylon) [inline packages/core-worker/director.js:167]: I'd go the other way round, default startDate to new Date(0) and then check if endDate is set or not, it makes it more clear for developers that the query will always fetch from the beginning if no explicit end or start date is set, an end date of new Date() could interfere with minor time differences, polling logic and such and throw up questions if that is safe to do so even though it is.

**Review by pozylon — CHANGES_REQUESTED** [review]
Good work!

File: packages/api/schema/query.js [hunk 2]
```diff
@@ -291,6 +291,8 @@ export default [
         offset: Int = 0
         status: [WorkStatus] = [NEW]
         selectTypes: [WorkType!] = []
+        startDate: Date
```
[the remark this answers is not in the document]
Comment (Mikearaya) [reply packages/api/schema/query.js:294]: I agree it is more descriptive this way and we can use it as a standard on other range filters to.

**Comment by pozylon** [comment]
@Mikearaya Ready for another review?

**Review by pozylon — APPROVED** [review]

**Pull request merged**